packages/presentation/lib/screen/cast_list/cast_crew_screen.dart

```diff
@@ -0,0 +1,133 @@
+              return Scaffold(
+                appBar: AppBar(
+                  backgroundColor: AppColorsDark.primaryColorDark,
+                  bottom: PreferredSize(
+                    preferredSize: const Size.fromHeight(Dimens.size1),
+                    child: Container(
+                      color: AppColorsDark.borderTabBar,
+                      height: Dimens.size1,
+                    ),
+                  ),
+                  elevation: Dimens.size0,
+                  title: Padding(
+                    padding: const EdgeInsets.only(
+                      left: Dimens.size12,
+                    ),
+                    child: Text(
+                      SM.current.castAndCrew,
+                      style: AppTextStyles.sfProSemiBold24px,
+                    ),
+                  ),
+                ),
+                body: Scrollbar(
+                  child: Padding(
+                      padding: const EdgeInsets.only(
+                        top: Dimens.size10,
+                        right: Dimens.size18,
+                        left: Dimens.size17,
+                      ),
+                      child: Responsive.isMobile(context)
+                          ? MovieListActors(
+                              cast: blocData.detailsAboutPeople,
+                              listLength: castLength!,
+                              isScrollable: true,
+                              additionalIndex: 0,
+                            )
+                          : ScrollConfiguration(
+                              behavior: ScrollConfiguration.of(context)
+                                  .copyWith(scrollbars: false),
+                              child: SingleChildScrollView(
+                                child: Row(
+                                  children: [
+                                    SizedBox(
+                                      width: width / 2.2,
+                                      child: MovieListActors(
+                                        cast: blocData.detailsAboutPeople,
+                                        listLength: halfCastLength,
+                                        isScrollable: false,
+                                        additionalIndex: 0,
+                                      ),
+                                    ),
+                                    const SizedBox(width: Dimens.size10),
+                                    SizedBox(
+                                      width: width / 2.2,
+                                      child: MovieListActors(
+                                        cast: blocData.detailsAboutPeople,
+                                        listLength: halfCastLength,
+                                        isScrollable: false,
+                                        additionalIndex: halfCastLength,
+                                      ),
+                                    ),
```

Comment:
use a gridview widget or better and end up responsive layout, if the app is fullscreen it's unlikely you'll be fine.